src/main/webapp/app/shared/subject/subject-pair-dialog.component.html

```diff
@@ -10,10 +10,28 @@
     <div class="modal-body">
         <div class="form-group">
             <label for="login" class="form-control-label"
-                [translate]="'managementPortalApp.subject.login'">
+                   [translate]="'managementPortalApp.subject.login'">
             </label>
             <input id="login" type="text" class="form-control" name="login" #loginInput="ngModel"
                    [(ngModel)]="subject.login" required minlength=1 maxlength=50 pattern="^[_'.@A-Za-z0-9- ]*$" [disabled]="subject.id">
+
+            <label for="externalId" class="form-control-label"
+                   [translate]="'managementPortalApp.subject.externalId'">
+            </label>
+            <input id="externalId" type="text" class="form-control" name="externalId" #loginInput="ngModel"
+                   [(ngModel)]="subject.externalId" required minlength=1 maxlength=50 pattern="^[_'.@A-Za-z0-9- ]*$" [disabled]="subject.id">
+
+            <label for="dateOfBirth" class="form-control-label"
+                   [translate]="'managementPortalApp.subject.dateOfBirth'">
+            </label>
+            <input id="dateOfBirth" type="text" class="form-control" name="dateOfBirth" #loginInput="ngModel"
+                   [(ngModel)]="subject.dateOfBirth" required minlength=1 maxlength=50 pattern="^[_'.@A-Za-z0-9- ]*$" [disabled]="subject.id">
+
+            <label for="name" class="form-control-label"
+                   [translate]="'managementPortalApp.subject.personName'">
+            </label>
+            <input id="name" type="text" class="form-control" name="name" #loginInput="ngModel"
+                   [(ngModel)]="subject.personName" required minlength=1 maxlength=50 pattern="^[_'.@A-Za-z0-9- ]*$" [disabled]="subject.id">
```

Comment:
Probably the input requirements can be removed (`required minlength=1 maxlength=50 pattern="^[_'.@A-Za-z0-9- ]*$"`)

Reply:
Would it make sense to only show these optional components if they have a non-null / non-empty value? 

I would expect person name before birth date.

Reply:
agreed and implemented. removed the input requirements from the existing component (subject.login) as well.